invenio_communities/views/communities.py

```diff
@@ -391,19 +410,22 @@ def communities_settings_privileges(pid_value, community, community_ui):
 
 
 @pass_community(serialize=True)
-def communities_settings_curation_policy(pid_value, community, community_ui):
-    """Community settings/curation-policy page."""
+def communities_settings_submission_policy(pid_value, community, community_ui):
+    """Community settings/submission-policy page."""
     permissions = community.has_permissions_to(PRIVATE_PERMISSIONS)
     if not permissions["can_update"]:
         raise PermissionDeniedError()
 
     return render_community_theme_template(
-        "invenio_communities/details/settings/curation_policy.html",
+        "invenio_communities/details/settings/submission_policy.html",
```

Comment:
```suggestion
        "invenio_communities/details/settings/record_submission_policy.html",
```

Reply:
Same here - this is the whole form template, which is called `Sybmission policy`